source/javascripts/all_nosearch.js

```diff
@@ -9,6 +9,12 @@ $(function() {
     window.recacheHeights();
     window.refreshToc();
   });
+
+  $('a').on('click', function() {
+    if ($(this).attr('href').startsWith('#')) {
+      gtag('config', gtagcode, { 'page_path': location.pathname + $(this).attr('href') });
```

Comment:
I don't know javascript well, trusting that you tested it and this references gtagcode properly :)